R/summarize_weighted_production.R

```diff
@@ -70,7 +70,7 @@ summarize_unweighted_production <- function(data, ...) {
       .data$loan_size_outstanding
     )) %>%
     distinct() %>%
-    group_by(...) %>%
+    group_by(.data$sector_ald, .data$technology, .data$year, ...) %>%
```

Comment:
This change was just to align the interface of `summarize_weighted_production` and `summarize_unweighted_production`. Both functions now automatically group by `sector_ald, technology and year`, regardless of input.